SCF-Session-4/split/SplitArrayTest.java

```diff
@@ -0,0 +1,36 @@
+package split;
+
+import static org.junit.Assert.*;
+
+public class SplitArrayTest 
+{
+	SplitArray splitArray = new SplitArray();
+	
+	@org.junit.Test
+	public void testPositive() 
+	{	
+		int indexSplit = splitArray.split(new int[]{1, 1, 1, 2, 1}, 5);
+		assertEquals(3, indexSplit);
+	}
+	
+	@org.junit.Test
+	public void secondTestNegative() 
+	{	
+		int indexSplit = splitArray.split(new int[]{1, 1, 1, 2, 1}, 5);
+		assertNotEquals(2, indexSplit);
+	}
+	
+	@org.junit.Test
+	public void testNegative()
+	{
+		int indexSplit = splitArray.split(new int[]{2, 1, 1, 2, 1}, 5);
+		assertEquals(-1, indexSplit);
+	}
+	
+	@org.junit.Test(expected = AssertionError.class)
+	public void testException()
+	{
+		int indexSplit = splitArray.split(new int[]{}, 0);
```

Comment:
Variable `indexSplit` is not used after declaration. Why it is declared?

Reply:
done